### ISSUE TITLE
find nail polish bottles for Oil Testing Kit

### ISSUE THREAD

**Issue by jywarren** [publiclab/publiclab-kits#7]
@steviepubliclab is going to try some out at the meetup in New Orleans on July 31

**Comment by Nataliegurl** [comment]
I will do some more sourcing if @steviepubliclab wants to pick something, then I can make the order when we are ready.

**Comment by ebarry** [comment]
I still have a couple hundred of the original bottles we used in NYC

**Comment by steviepubliclab** [comment]
@jywarren should we try to get some of Liz's bottles down to NOLA or do you want us to order new ones? If so I think I'll work with @Nataliegurl to order these in the Rhone:
http://www.ebottles.com/showbottles-bottle-1329-gclid-Cj0KEQjwopOeBRC1ndXgnuvx8JYBEiQAq4RPt_MBIoRstpRXJzK66iXV1lbaZ6J7TWOot8oR6wDas7QaAops8P8HAQ-kw-NAIL_POLISH_BOTTLES_WITH_BRUSHES___GLASS.htm

**Comment by Nataliegurl** [comment]
I don't know anything about the bottles Liz has except how they look. Where
did they come from and how much are they? Did people not want to keep using
them?

**Comment by jywarren** [comment]
They do not have brushes in their caps, and that's one major refinement
we're looking to add to the kits. It'd be nice to try to find brush caps
that fit them, but if we only have a few hundred left, we'd just have to do
another search later, and we really need bottles that we'll be shipping in
kits before too long. If we find bottles whose caps fit the old bottles,
that's great, but I don't think we can let that slow our sourcing process
down.

**Comment by steviepubliclab** [comment]
@Nataliegurl lets just plan on ordering some of those I posted for us to try in the workshop. Do you do this, or should I? Either works for me.

**Comment by Nataliegurl** [comment]
That's fine, but if we find brush caps, can we reorder the bottles?

**Comment by jywarren** [comment]
There are none left from the source Liz had - she bought the last ones. I think that's a sourcing dead end :-(

**Comment by Nataliegurl** [comment]
Check.

**Comment by jywarren** [comment]
Repeating here from a lost email thread: brushes in caps could either:

a) be broken off once you've got the sample in the jar
b) be shortened at factory or in Portland so they only dip in the top of the mineral oil

**Comment by steviepubliclab** [comment]
I'm not sure this is easier than just using cotton swabs.

**Comment by Nataliegurl** [comment]
I agree. It's also more work, time, an overhead. Are the brushes necessary since they are being disposed of?

**Comment by jywarren** [comment]
It definitely would be easier and would reduce toxic waste to leave them in the bottles. Swabs get dirty or broken easily and people are inclined to use them more than once. Also the cotton rubs off a bit and stiff brushes may work better. And the cotton may have brightening agents in it, affecting the fluorescence data. This is something we've really wanted to do for a while based on running sample prep workshops.

**Comment by Nataliegurl** [comment]
But you cannot reuse the brushes right? Don't you risk cross contamination?

**Comment by jywarren** [comment]
Correct, by having a brush in each bottle, they will only be used once per
sample.

**Comment by Nataliegurl** [comment]
Did we agree on cuvettes? Can we close this and I'll source cuvettes instead?
